package.json

```diff
@@ -52,6 +52,7 @@
     "@types/node": "22.14.1",
     "@types/react": "19.1.2",
     "@types/react-dom": "19.1.2",
+    "dotenv": "^16.5.0",
```

Comment:
I was getting
```
➜  ai-movie-mashup git:(main) pnpm prisma generate
Failed to import config file as TypeScript from "/Users/justin/rw/blotter/ai-movie-mashup/prisma.config.ts". Error: Error: Cannot find module 'dotenv/config'
```

Reply:
oh. I was testing `prisma.config.ts` for their d1 migrations -- but not needed. I'll cleanup. thnx!